src/services/users.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+let users = [];
+
+function reset() {
+  users = [];
+}
+
+function getAll() {
+  return users;
+}
+
+function getById(id) {
+  const foundUser = users.find(user => user.id === id);
+
+  return foundUser || null;
+}
+
+function create(name) {
+  const newUser = {
+    id: Math.floor(Math.random() * 100),
```

Comment:
This will cause duplicate ids sooner or later. Instead, you can make a variable at the top of this file, near `let users = [];`, that keeps track of the latest id. After that you can just do this:
```suggestion
    id: latestId++,
```

Real databases have similar functionality used for simple integer ids (for example in [postgres](https://chartio.com/resources/tutorials/how-to-define-an-auto-increment-primary-key-in-postgresql/))